test/data/brave_scheme_load.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<script>
+  const url = "brave://settings"
+  function openBraveSettings() {
+    window.open(url);
```

Comment:
[minor] this is already blocked currently, though it doesn't throw a console error AFAIK. `window.open('brave://settings', '', 'noopener')` is not blocked

Reply:
Oh, I forgot to add test for `window.open` with  `noopener` case. `NotAllowedToLoadTestByWindowOpenWithNoOpener` case is added for that.

Right. `window.open(url)` was blocked before but blocking timing is different with latest.
Previously, this navigation is requested from renderer and browser mapped it to chrome scheme and renderer blocked it.
For now, renderer initially block it.